plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeLocalConcurrentWritesTest.java

```diff
@@ -233,11 +233,11 @@ private void testConcurrentInsertsSelectingFromTheSameTable(boolean partitioned)
                                     .map("(%d, 10)"::formatted)
                                     .collect(joining(", ", ", ", "")));
             assertQuery(
-                    "SELECT version, operation, isolation_level, read_version FROM \"" + tableName + "$history\"",
```

Comment:
`testConcurrentInsertsSelectingFromTheSameTable` is copied from `BaseDeltaLakeConnectorSmokeTest`. Why not fix the original test as well? 

Same for testConcurrentInsertsReconciliationForMixedInserts.